Update active-directory-conditional-access-technical-reference.md

spunukol opened the pull request:
Removed "preview" for Chrome. Updated the Note for Chrome Extension on the build versions

azurecla commented:
Hi __@lucaslin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

spunukol committed:
Update active-directory-conditional-access-technical-reference.md

Removed "preview" for Chrome. Updated the Note for Chrome Extension on the build versions
